index.html

```diff
@@ -1307,7 +1307,7 @@ <h4 class="footer-links">Legal</h4>
                 height="40"
               />
             </a>
-            <div class="footer-links">
+            <div style="text-align:center;" class="footer-links">
```

Comment:
You should not use inline styles

Reply:
yes that i should not do this but you can easily write that property in you "footer-links" class and it will work